Can't build GNU Emacs or Remacs on macOS 10.14 Mojave - fatal error: 'libxml/tree.h' file not found

Might be off-topic but I get this error on macOS 10.14 Mojave, after running `make`:

`xml.c:23:10: fatal error: 'libxml/tree.h' file not found`

Both `xcode-select --install` and `brew install libxml2` is installed. I get the same error on GNU Emacs as well. Do you get this error or do you know about a solution to it?

I didn't get this error before updating macOS to Mojave.

--- a/README.md
+++ b/README.md
@@ -141,21 +141,20 @@ more Emacs-y.
    something like `apt install build-essential automake clang`. On
    macOS, you'll need Xcode.
 
-3. You will need some C libraries. On Linux, you can install
-   everything you need with:
+3. Linux:
 
         apt install texinfo libjpeg-dev libtiff-dev \
           libgif-dev libxpm-dev libgtk-3-dev libgnutls28-dev \
           libncurses5-dev libxml2-dev libxt-dev
 
-    On macOS, you'll need libxml2 (via `xcode-select --install`) and
-    gnutls (via `brew install gnutls`).
-
-    On macOS, the default `makeinfo` command is outdated, you'll need
-    to update it (via `brew install texinfo`). To use the installed
-    version of `makeinfo` instead of the built-in (`/usr/bin/makeinfo`)
-    one, you'll need to make sure `/usr/local/opt/texinfo/bin` is
-    before `/usr/bin` in `PATH`.
+   MacOS:
+   
+        brew install libxml2 gnutls texinfo
+        
+    To use the installed version of `makeinfo` instead of the built-in 
+    (`/usr/bin/makeinfo`) one, you'll need to make sure `/usr/local/opt/texinfo/bin` 
+    is before `/usr/bin` in `PATH`.
+    Mojave: `export PKG_CONFIG_PATH="/usr/local/opt/libxml2/lib/pkgconfig"`
 
 ## Dockerized development environment
 

